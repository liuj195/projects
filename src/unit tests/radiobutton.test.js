import React from "react";
import RadioButton from "../components/buttons/radiobutton";
import renderer from "react-test-renderer";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

test("RadioButton renders correctly", () => {
  const fakeFunction = function() {};
  const component = renderer.create(
    <RadioButton
      onChange={fakeFunction}
      style={{ display: "inline-block" }}
      id="Single Window"
      name="windowSelect"
      value="Single Window"
      checked={true}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
