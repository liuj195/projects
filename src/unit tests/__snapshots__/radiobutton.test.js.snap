// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton renders correctly 1`] = `
<li
  style={
    Object {
      "display": "inline-block",
    }
  }
>
  <input
    checked={true}
    id="Single Window"
    name="windowSelect"
    onChange={[Function]}
    type="radio"
    value="Single Window"
  />
  <label
    htmlFor="Single Window"
  >
    Single Window
  </label>
</li>
`;
